app/assets/stylesheets/_global.scss

```diff
@@ -377,3 +377,9 @@ a[title="Back"] {
 .faded {
   color: $gray3;
 }
+input[type=date].filled::-webkit-datetime-edit { 
```

Comment:
What's the reason for this webkit-specific selector? And what's the effect in non-webkit browsers?

Reply:
The input date behaves differently in each type of browser and it doesn't obey the placeholder coloring, is a kind of input that the browser builds (I believe it also depends on the operating system). I found several attempts to solve, I tried only with CSS, but in the end, I needed a little jquery too. CSS selectors to change color like `:empty` or `:placeholder-shown` doesn't work on this type of input. Placeholder doesn't work too. It is possible that this has problems in other browsers as well, I forgot to test it on Firefox and maybe someone could test on Edge. This problem was happening on every input date in the system, not just on this screen.

Reply:
I didn't find how to do this on Mozilla and Edge. And I can't use placeholder, it doesn't work in Chrome, and if it works on the other browsers, it's not recommended, because the date format depends of user's operating system. So, if I put DD/MM/YYYY in placeholder and user's system is in English, it will fill MM/DD/YYYY.

Reply:
@straight-shoota could I mark this issue as resolved?